.github/workflows/aur.yml

```diff
@@ -0,0 +1,42 @@
+name: aur
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  aur-publish:
+    name: Publish to AUR
+    runs-on: ubuntu-latest
+    container: archlinux/base
+
+    steps:
+      # see: https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/m-p/32167#M1027
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Publish
+        run: |
+          pacman -Sy --needed --noconfirm sudo git binutils openssh && \
+          useradd builduser -m && \
+          su builduser -c "
+            mkdir ~/.ssh && \
+            echo \"${{ secrets.SSH_PUBLIC_KEY }}\" > ~/.ssh/id_rsa.pub && \
+            echo \"${{ secrets.SSH_PRIVATE_KEY }}\" > ~/.ssh/id_rsa && \
+            chmod 600 ~/.ssh/id_rsa && \
+            ssh-keyscan aur.archlinux.org >> ~/.ssh/known_hosts && \
+            git config --global user.email \"work@mgrachev.com\" && \
+            git config --global user.name \"Dotenv Linter\" && \
+            cd ~ && \
+            git clone ssh://aur@aur.archlinux.org/dotenv-linter-bin.git && \
+            cd dotenv-linter-bin && \
+            sed -i \"s/pkgver=.*/pkgver=${{ steps.get_version.outputs.VERSION }}/\" PKGBUILD && \
+            makepkg --printsrcinfo > .SRCINFO
+            git status && \
+            git diff && \
```

Comment:
Are these commands needed for debug?

Reply:
I've decided to let them in for the first one or two releases to see immediately what went wrong if some error occurs and have easier debugging if you think this is useless noise and shouldn't be there until we have a problem I can remove it.

Reply:
Okay, let's leave as is.